show how to export to CSV and XLSX files

fangohr opened the pull request:
- needs openpyxl library for export to excel

fangohr committed:
show how to export to CSV and XLSX files

- needs openpyxl library for export to excel

kirienko reviewed (approved):
LGTM.

One possible improvement is to say explicitly something like «_this file will be saved on you disk [somewhere-I-don't-know-where]_». Now we broadly say "export" without any details. 

BTW, how does this saving process work in Binder? Is there a saving dialogue or something? (Please forgive me my laziness :smiley:)

fangohr committed:
more details on saving  files added

- files are saved to local directory
- warning about binder

- also add link to Pandas documentation
